src/Functions.Worker.Extensions.OpenAI/OpenAIModels.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.OpenAI;
+
+static class OpenAIModels
+{
+    // Reference - https://platform.openai.com/docs/models
+
+    /// <summary>
+    /// GPT 3 Turbo, refer to https://platform.openai.com/docs/models/continuous-model-upgrades for exact model being pointed to
+    /// </summary>
+    internal const string Gpt_35_Turbo = "gpt-3.5-turbo";
```

Comment:
I suggest we instead name this `DefaultChatModel`, similar to how we named the property below as `DefaultEmbeddingsModel`. We'll need to update the comment accordingly.

Reply:
done